pkg/pillar/zedUpload/datastore.go

```diff
@@ -26,6 +26,10 @@ const (
 	SyncOpDownloadWithSignature = 4
 	SyncOpList                  = 5
 	SyncOpGetObjectMetaData     = 6
+	SyncOpGetURI                = 7
+	SyncOpDownloadByChunk       = 8
```

Comment:
We don't need new SyncOp for download. We are currently downloading in chunks. You can use an existing API for your use case.